### PR TITLE
Add remaining stories

### PR DESCRIPTION
**Pull request by DawoodShahat**
Added Stories for:

- ```FilterInput.svelte``` component
- ```Pill.svelte``` component

### CODE REVIEW

File: stories/filterinput.stories.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import FilterInput from "../src/components/FilterInput.svelte";
+
+export default {
+  title: "FilterInput",
+};
+
+export const Basic = () => ({
+  Component: FilterInput,
+  props: "",
+});
```
Comment (wlach) [inline stories/filterinput.stories.js:10]: Can you add some examples of a simple list of items being filtered? e.g.

"foo", "bar", "foobar"

**Review by wlach — CHANGES_REQUESTED** [review]
Thanks! I think we could flesh out the FilterInput example a bit more. Note that you can create components especially for telling stories, see e.g.

https://github.com/graph-paper-org/graph-paper/tree/main/packages/button/stories

File: stories/filterinput.stories.js (hunk 1 above)
Comment (DawoodShahat) [reply stories/filterinput.stories.js:10]: Alright. Adding in the next commit.

**Comment by DawoodShahat** [comment]
upon @wlach review
- added a list of items to be filtered by FilterInput component
- added knobs to play with both Pill and FilterInput component interactively

**Comment by wlach** [comment]
Thanks @DawoodShahat!

**Pull request merged**